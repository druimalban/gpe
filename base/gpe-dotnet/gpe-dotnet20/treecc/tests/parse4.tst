// test type names
%node x %typedef =
{
	[
	int x;
	const char *y;
	int;
	Element[][] z;
	const Element[]*[] w;
	char[ v;
	char[[* u;
	unsigned int t;
	object& obj;
}
