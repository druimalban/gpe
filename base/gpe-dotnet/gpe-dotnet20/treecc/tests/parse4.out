line 2: %node x no_parent 4
parse4.tst:4: type name expected
parse4.tst:4: field declaration expected
parse4.tst:5: `;' expected
line 5: %field x int no_value 0
line 6: %field y const char * no_value 0
parse4.tst:7: field name expected
line 8: %field z Element[][] no_value 0
line 9: %field w const Element[] *[] no_value 0
parse4.tst:10: `]' expected
line 10: %field v char[] no_value 0
parse4.tst:11: `]' expected
parse4.tst:11: `]' expected
line 11: %field u char[][] * no_value 0
line 12: %field t unsigned int no_value 0
line 13: %field obj object & no_value 0
